packages/dx-react-scheduler-material-ui/src/templates/appointment-form/all-day-editor.test.jsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { createShallow } from '@material-ui/core/test-utils';
+import { AllDayEditor } from './all-day-editor';
+
+describe('Appointment Form', () => {
+  let shallow;
+  beforeAll(() => {
+    shallow = createShallow({ dive: true });
+  });
+  describe('AllDayEditor', () => {
+    it('should pass rest props to the root element', () => {
```

Comment:
The test name is 'should pass rest props to the root element' but you check the 'className' prop.

Reply:
Yeah, this component has not a `className` property. In this case, `className` will be spread with `restProps`.

Reply:
I see. OK, let's keep it as is.